test/camunda-cloud/features/modeling/CleanUpBusinessRuleTaskBehaviorSpec.js

```diff
@@ -0,0 +1,429 @@
+import {
+  bootstrapCamundaCloudModeler,
+  inject
+} from 'test/TestHelper';
+
+import {
+  is
+} from 'bpmn-js/lib/util/ModelUtil';
+
+import { getBusinessObject } from 'bpmn-js/lib/util/ModelUtil';
+
+import diagramXML from './process-businessRuleTask.bpmn';
+
+
+describe('camunda-cloud/features/modeling - CleanUpBusinessRuleTaskBehavior', function() {
+
+  beforeEach(bootstrapCamundaCloudModeler(diagramXML));
+
+
+  describe('triggered via propertiesPanel', function() {
```

Comment:
Although the test cases are already pretty complete (👍 ), shall we maybe add some when the behaviors don't get triggered? E.g. when

* the element is no business rule task
* when other extension elements (as inputs, outputs) got added

Wdyt?

Reply:
Sure, but these will never be complete obviously. I will add some basic example to validate the basic functionality :+1: